app/classifier/task-nav.jsx

```diff
@@ -105,27 +112,26 @@ class TaskNav extends React.Component {
   }
 
   render() {
-    if (!this.props.annotation) {
-      return null;
-    }
+    const completed = !!this.props.classification.completed;
+
     const task = this.props.task ? this.props.task : this.props.workflow.tasks[this.props.workflow.first_task];
```

Comment:
I'm not 100% sure, but removing this might break the classifier for some types of tasks. It forces the component to wait until the classification, task and annotations have initialised before rendering.